README.md

```diff
@@ -362,6 +363,13 @@ if a playlist is blacklisted, it will be blacklisted for a period of that
 customized duration. This enables the blacklist duration to be configured
 by the user.
 
+#### maxPlaylistRetries
+* Type: `number`
+* can be used as an initialization option
```

Comment:
Would be good to specify that the default for this is `Infinity`

Reply:
👍